source/clear-linux/get-started/virtual-machine-install/vmw-player.rst

```diff
@@ -71,19 +62,14 @@ Install the VMware Workstation Player hypervisor
 
 For additional help, see the `VMware Workstation Player guide`_.
 
-|CL| image types
-****************
-
-.. include:: ../../reference/image-types.rst
-   :Start-after: image-types-content:
-
-
 Download the latest |CL| installer ISO
 **************************************
 
 Get the latest |CL| installer ISO image from the `image`_ repository.
 Look for :file:`clear-[version number]-installer.iso.xz`.
 
+Visit :ref:`image-types` for additional information about all available |CL| images.
```

Comment:
At the start of line 56 and 60, please fix the formatting errors. Check that these lines are left-aligned with the line that precedes them.

Reply:
Good catch!